.github/workflows/ci.yml

```diff
@@ -18,14 +18,18 @@ jobs:
       contents: read  # for actions/checkout to fetch code
     name: "${{ matrix.root-pom }} on JDK ${{ matrix.java }} on ${{ matrix.os }}"
     strategy:
+      fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        java: [ 8, 11, 17 ]
+        java: [ 8, 11, 17, 21 ]
         root-pom: [ 'pom.xml', 'android/pom.xml' ]
         include:
           - os: windows-latest
             java: 17
             root-pom: pom.xml
+          - os: windows-latest
+            java: 21
```

Comment:
For Windows, I think I'll just switch us to 21 (i.e., remove 17), updating the branch-protection rules around the same time.

(Even with performance improvements possibly on the horizon, I worry about having more CI jobs. My hope would be that one Windows job is enough: The main thing is to avoid embarrassing "Windows does not work" bugs like the one I introduced not so long ago.)

Reply:
Roger that